ui/legacy/components/cstudio-forms/data-sources/components.js

```
@@ -58,6 +58,22 @@
+					const unsubscribe = store.subscribe(() => {
+						if (store.getState().contentTypes?.byId) {
+							unsubscribe();
+							this._renderControlEntries(control);
+						}
+					});
+				}
+			});
+		},
+
+		_renderControlEntries: function (control) {
+			debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)